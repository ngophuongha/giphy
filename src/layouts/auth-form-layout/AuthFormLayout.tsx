import { AuthFormContainer, AuthFormWrapper } from "./AuthFormLayout.styled";

export const AuthFormLayout: React.FC = ({ children }) => {
  return (
    <AuthFormWrapper>
      <AuthFormContainer>{children}</AuthFormContainer>
    </AuthFormWrapper>
  );
};
