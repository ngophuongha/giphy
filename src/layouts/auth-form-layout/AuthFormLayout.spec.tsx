import { render } from "../../../test";
import { AuthFormLayout } from "./AuthFormLayout";

describe("Auth Form Layout", () => {
  it("should match snapshot", () => {
    const { container } = render(<AuthFormLayout />);
    expect(container).toMatchSnapshot();
  });
});
