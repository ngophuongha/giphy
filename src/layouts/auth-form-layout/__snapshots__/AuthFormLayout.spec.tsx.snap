// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth Form Layout should match snapshot 1`] = `
.c0 {
  background-color: #fff;
  color: #000000;
  overflow-x: hidden;
}

.c1 {
  height: 100vh;
  width: 100vw;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: auto;
  background-image: linear-gradient( to right, #a1343e, #2b5dab );
}

.c1 .form-container {
  height: 70%;
  width: 100%;
  margin: auto;
}

.c2 {
  width: 90%;
  margin: auto;
  border-radius: 0.25rem;
  padding: 3rem;
  background-color: #fff;
  color: #1b1c1c;
}

@media only screen and (min-width:768px) {
  .c2 {
    width: 50%;
  }
}

@media only screen and (min-width:1024px) {
  .c2 {
    width: 40%;
  }
}

@media only screen and (min-width:1280px) {
  .c2 {
    width: 25%;
  }
}

<div>
  <div
    class="c0 c1"
  >
    <div
      class="c2"
    />
  </div>
</div>
`;
